recipes/pygaul/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pygaul" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygaul-{{ version }}.tar.gz
+  sha256: a98b879379825e699ab553cb0874f17ce87b749720f1c508f635110889938a98
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.2
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - deprecated >=1.2.14
+    - pandas
+    - earthengine-api
+    - pyarrow
+
+test:
+  imports:
+    - pygaul
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pygaul/
+  summary: Easy access to administrative boundary defined by FAO GAUL from a Python script.
```

I think it would be nice to add a link to the GitHub repo:

```suggestion
  summary: Easy access to administrative boundary defined by FAO GAUL from a Python script.
  dev_url: https://github.com/12rambau/pygaul
```